recipes/pyspark/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "pyspark" %}
+{% set version = "2.1.0" %}
+{% set hadoop_version = "2.7" %}
+{% set sha256 = "0834c775f38473f67cb122e0ec21074f800ced50c1ff1b9e37e222a0069dc5c7" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: http://www.apache.org/dist/spark/spark-{{ version }}/spark-{{ version }}-bin-hadoop{{ hadoop_version }}.tgz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: >
+    cd python &&
+    python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - pip
+    - pypandoc
+    - py4j 0.10.4
+    - setuptools
+
+  run:
+    - python
+    - numpy >=1.7
+    - pandas
+    - py4j 0.10.4
+
+test:
+  imports:
+    - pyspark
+    # - pyspark.mllib # Subpackages of this missing in 2.1.0 release
+    # - pyspark.ml    # Ditto
+    - pyspark.sql
+    - pyspark.streaming
+
+about:
+  home: http://spark.apache.org/
+  license: Apache
```

Comment:
I can update. Can I mark it [ci skip] or does it have to go through all the CI builds again?